### ISSUE TITLE
Didn't get the module to work

### ISSUE THREAD

**Issue by iComputerfreak** [fantasycalendar/FoundryVTT-RestRecovery#9]
I don't know what I'm doing wrong, but I tried for a good 30 minutes now to get your module working.

I created a new world with the D&D 5e system, created a new, empty scene, added the Wizard starter hero from the SRD Compendium, set the available spell slots to 0/2 and then took a short rest, but the short rest prompt is still the default one.

The actor has the "Arcane Recovery" feature and is a Wizard level 1 with 1/1 uses of Arcane Recovery.
I only have your module and libWrapper installed at their newest versions and default configurations using Foundry 0.8.9

Is there anything special one has to do or activate?

I really am out of ideas what I could be doing wrong. I also have no errors or warnings in my console.

**Comment by iComputerfreak** [comment]
According to D&D Beyond, it is a 1st level feature: https://www.dndbeyond.com/classes/wizard#TheWizardTable

**Comment by Haxxer** [comment]
Could you try this in 1.0.0 and see if it's still broken?

**Comment by iComputerfreak** [comment]
Yes. I will update the module and report back, if it persists.

**Comment by Haxxer** [comment]
Could you export the wizard character and I'll take a look on my end?

**Comment by iComputerfreak** [comment]
I think you made the same level-2-mistake in your code: https://github.com/fantasycalendar/FoundryVTT-RestRecovery/blob/891dc00f2b0ccd27e98eaa1d2baca5cc58846248/scripts/rest-workflow.js#L107

If I set my wizard level ingame to 2, Arcane Recovery seems to work correctly.

The issue that I get all spell slots back, if I'm level 1 or a Sorcerer seems to be separate.

**Comment by Haxxer** [comment]
Yeah, that's one of the issues - the other is that even if I patch the item to have 1/1 long rest uses, the following code doesn't take that patch into account. I have fixed it locally, will release a patch tonight.

**Comment by iComputerfreak** [comment]
Thank you. I will take a look at it again tomorrow then.

**Comment by DarKDinDoN** [comment]
Hello,

I confirm that taking a short rest with or without a special feature like "Arcane recovery" will replenish the spell slots to the maximum.

**Comment by Haxxer** [comment]
I have also localized this bug and fixed it

**Comment by iComputerfreak** [comment]
Everything seems to work as expected now on my end :+1:
